addons/interaction/CfgVehicles.hpp

```diff
@@ -435,7 +435,7 @@ class CfgVehicles {
             class ACE_MainActions: ACE_MainActions {
                 class GVAR(flip) {
                     displayName = CSTRING(Flip);
-                    condition = QUOTE(call DFUNC(canFlip));
+                    condition = QUOTE(_target call DFUNC(canFlip));
```

Comment:
I guess a raw `call FUNC` is going to be a tiny bit faster than `_var call FUNC`
but it really shouldn't matter if we think this is easier to read

Reply:
Raw is actually consistently slower when I tested, even an empty array is somehow faster than giving nothing.

Reply:
```
_target = 5; 
[_target] call { 
    call f; x20
0.0136198 ms


_target = 5; 
[_target] call { 
    _target call f; x20
0.0173643 ms


_target = 5; 
[_target] call { 
    [_target] call f;  x20
0.0179958 ms
```

Reply:
Personally I find it's easier to read, but I'll revert it and update the header instead.